algorithms/src/std_algorithms/impl/Kokkos_Constraints.hpp

```diff
@@ -29,7 +29,7 @@ struct is_admissible_to_kokkos_std_algorithms : std::false_type {};
 
 template <typename T>
 struct is_admissible_to_kokkos_std_algorithms<
-    T, std::enable_if_t< ::Kokkos::is_view<T>::value && T::rank == 1 &&
+    T, std::enable_if_t< ::Kokkos::is_view<T>::value && T::rank() == 1 &&
```

Comment:
MSVC was choking on that guy.  Was giving failed static assertions all over the place in algorithms